webtau/src/main/java/com/twosigma/webtau/expectation/SystemTimerConfig.java

```diff
@@ -21,20 +21,22 @@
 import com.twosigma.webtau.cfg.WebTauConfig;
 
 public class SystemTimerConfig implements ExpectationTimerConfig {
-    private static WebTauConfig cfg = WebTauConfig.INSTANCE;
-
     @Override
     public ExpectationTimer createExpectationTimer() {
         return new SystemTimeExpectationTimer();
     }
 
     @Override
     public long defaultTimeoutMillis() {
-        return cfg.waitTimeout();
+        return getCfg().waitTimeout();
     }
 
     @Override
     public long defaultTickMillis() {
         return 100;
     }
+
+    private WebTauConfig getCfg() {
```

Comment:
`getCfg()` is sometimes static, sometimes not, feels like it should be consistent.

Reply:
Logic was to use static for classes with static methods, and non static otherwise. Your other suggestion is good. So I am renaming to getCfg.